scapy/layers/netflow.py

```diff
@@ -101,3 +133,316 @@ class NetflowRecordV5(Packet):
+        29: 1,
+        30: 1,
+        31: 3,
+        32: 2,
+        33: 1,
+        34: 4,
+        35: 1,
+        36: 2,
+        37: 2,
+        38: 1,
+        39: 1,
+        40: 4,
+        41: 4,
+        42: 4,
+        46: 1,
+        47: 4,
+        48: 1,
+        49: 1,
+        50: 4,
+        55: 1,
+        56: 6,
+        57: 6,
+        58: 2,
+        59: 2,
+        60: 1,
+        61: 1,
+        62: 16,
+        63: 16,
+        64: 4,
+        70: 3,
+        71: 3,
+        72: 3,
+        73: 3,
+        74: 3,
+        75: 3,
+        76: 3,
+        77: 3,
+        78: 3,
+        79: 3,
+    }
+
+class NetflowHeaderV9(Packet):
+    name = "Netflow Header V9"
+    fields_desc = [ ShortField("count", 0),
+                    IntField("sysUptime", 0),
+                    UTCTimeField("unixSecs", 0),
+                    IntField("packageSequence",0),
+                    IntField("SourceID", 0) ]
+
+class NetflowTemplateFieldV9(Packet):
+    name = "Netflow Flowset Template Field V9"
+    fields_desc = [ ShortEnumField("fieldType", None, NetflowV9TemplateFieldTypes),
+                    ShortField("fieldLength", 0) ]
+    def __init__(self, *args, **kwargs):
+        Packet.__init__(self, *args, **kwargs)
+        if self.fieldType != None:
+            self.fieldLength = NetflowV9TemplateFieldDefaultLengths[self.fieldType]
+
+    def default_payload_class(self, p):
+        return conf.padding_layer
+
+class NetflowTemplateV9(Packet):
+    name = "Netflow Flowset Template V9"
+    fields_desc = [ ShortField("templateID", 255),
+                    FieldLenField("fieldCount", None, count_of="template_fields"),
+                    PacketListField("template_fields", [], NetflowTemplateFieldV9,
+                                    count_from = lambda pkt: pkt.fieldCount) ]
+
+    def default_payload_class(self, p):
+        return conf.padding_layer
+
+class NetflowFlowsetV9(Packet):
+    name = "Netflow FlowSet V9"
+    fields_desc = [ ShortField("flowSetID", 0),
+                    FieldLenField("length", None, length_of="templates", adjust=lambda pkt,x:x+4),
+                    PacketListField("templates", [], NetflowTemplateV9,
+                                    length_from = lambda pkt: pkt.length-4) ]
+
+class NetflowRecordV9(Packet):
+    name = "Netflow DataFlowset Record V9"
+    fields_desc = [ StrField("fieldValue", "") ]
+
+    def default_payload_class(self, p):
+        return conf.padding_layer
+
+class NetflowDataflowsetV9(Packet):
+    name = "Netflow DataFlowSet V9"
+    fields_desc = [ ShortField("templateID", 255),
+                    FieldLenField("length", None, length_of="records", adjust = lambda pkt,x:x+4),
+                    PadField(PacketListField("records", [], NetflowRecordV9,
+                                    length_from = lambda pkt: pkt.length-4),
+                             4, padwith=b"\x00") ]
+
+    @classmethod
+    def dispatch_hook(cls, _pkt=None, *args, **kargs):
```

Comment:
This **kargs should be **kwargs

Reply:
Please do not comment on closed pull requests but open an issue (or even better, a pull request).